source/game.programme.programmedata.bmx

```diff
@@ -1473,24 +1473,38 @@ Type TProgrammeData Extends TBroadcastMaterialSource {_exposeToLua}
 			'modifiers could increase or decrease influences of age/aired/...
 			Local ageInfluence:Float = 1.5 * age * GetModifier(modKeyTopicality_AgeLS)
 			Local timesBroadcastedInfluence:Float = timesBroadcasted * GetModifier(modKeyTopicality_TimesBroadcastedLS)
-			'by default first broadcast has no influence but programmes like sport matches
-			'should loose a big bit of max topicality after the first time
-			'on TV. Also they should loose topicality as soon as they are
-			'no longer "live" (after live time ended) and age faster
+
 			Local firstBroadcastInfluence:Float = 10 * (timesBroadcasted>0)
 			Local notLiveInfluence:Float = 0.0
 			If IsLiveOnTape()
+				'Live programmes like sport matches should lose attractiveness after the first broadcast.
 				notLiveInfluence = 10 * GetModifier(modKeyTopicality_NotLiveLS, 1.0)
+				firstBroadcastInfluence:* GetModifier(modKeyTopicality_FirstBroadcastDoneLS, 1.0)
+				'Also they should age much faster
 				weightAge = 1.0
-				firstBroadcastInfluence: * GetModifier(modKeyTopicality_FirstBroadcastDoneLS, 1.0)
+				Local daysSinceLive:Int = Max(0, GetWorldTime().GetDay() - GetWorldTime().GetDay(releaseTime))
```

Comment:
Einen kleinen Unterschied gibt es zwischen dem hiesigen
```
Local daysSinceLive:Int = Max(0, GetWorldTime().GetDay() - GetWorldTime().GetDay(releaseTime))
```

und dem Code vom "Sport":
```
Local daysSinceMatch:Int = Max(0, GetWorldTime().GetDay() - GetWorldTime().GetDay(endTime))
```

Ersteres nimmt die "Startzeit", letzteres die "Endzeit". Bei normalen "14-16" Uhr-Matches macht das keinen Unterschied. Ein Match von "22-02 Uhr" hingegen ist bei der Sportbegegnung dann 4 Uhr nachts noch "0 Tage", beim neuen Code allerdings schon "1 Tag".

Denke aber... das wird eher selten passieren, also kein grosses Ding.

Reply:
Diese Änderung hatte ich auch für vernachlässigbar gehalten. Wenn es irgendwann wichtig wäre, könnte man vermutlich auf die Release-Zeit noch die Anzahl der Blöcke in Stunden draufaddieren.